packages/pluggableWidgets/datagrid-web/src/components/Table.tsx

```diff
@@ -39,6 +39,7 @@ export interface TableProps<T> {
     data: T[];
     emptyPlaceholderRenderer?: (renderWrapper: (children: ReactNode) => ReactElement) => ReactElement;
     filterRenderer: (renderWrapper: (children: ReactNode) => ReactElement, columnIndex: number) => ReactElement;
+    filtersTitle?: string;
```

Comment:
Can we make the empty placeholder part also better accessible by giving it a title? Or is it accessible enough?

Reply:
Empty placeholder for Data grid is a cell. and not a section